RecoverCRPVM/ReadMe .md

```diff
@@ -0,0 +1,48 @@
+#Overview
+Occasionally Azure IaaS VMs (Microsoft.Compute/virtualMachines) may not start because there is something wrong with the operating system (OS) disk preventing it from booting up correctly.
+In such cases it is a common practice to recover the problem VM by performing the following steps:
+	-Stop VM
+	-Take a snapshot of the OS Disk
+	-Create a Temporary Rescue VM
+	-Attach the OS Disk to the Rescue VM
+	-RDP to RescueVM
+	-Run the script https://github.com/sebdau/azpstools/blob/master/FixDisk/TS_RecoveryWorker2.ps1 as an elevated administrator from the recovery VM and perform other manual steps
+	-Detach the Data disk from Rescue VM
+	-Perform Disk Swap to point the OsDisk.Vhd.Uri to the recovered OS Disk Uri
+        -Finally Remove all the resources that were created for the Rescue VM
+
+#Current version supports 
+    Microsoft.Compute/virtualMachines (Non-Managed VM's)
+    Supports Windows OS
+
+# Scenarios
+
+##  When would you use the script?
+If a Windows VM in Azure does not boot. Typically in this scenario VM screenshot from [boot diagnostics] (https://azure.microsoft.com/en-us/blog/boot-diagnostics-for-virtual-machines-v2/) 
```

Comment:
no space after [boot diagnostics]  it breaks the hyper link

Reply:
done